src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/azcopy.py

```diff
@@ -0,0 +1,68 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+from ..azcopy.util import AzCopy, blob_client_auth_for_azcopy, login_auth_for_azcopy
+
+
+def storage_blob_copy(azcopy, source, destination, recursive=None):
+    flags = []
+    if recursive is not None:
+        flags.append('--recursive')
+    azcopy.copy(source, destination, flags=flags)
+
+
+def storage_blob_upload(cmd, client, source, destination, recursive=None):
+    azcopy = _azcopy_blob_client(cmd, client)
+    storage_blob_copy(azcopy, source, _add_url_sas(destination, azcopy.creds.sas_token), recursive=recursive)
+
+
+def storage_blob_download(cmd, client, source, destination, recursive=None):
+    azcopy = _azcopy_blob_client(cmd, client)
+    storage_blob_copy(azcopy, _add_url_sas(source, azcopy.creds.sas_token), destination, recursive=recursive)
+
+
+# def storage_blob_upload_batch(cmd, client, source, destination):
```

Comment:
Any reason we comment it out here? Because we will use it later?

Reply:
Removed.